_includes/single-post.html

```diff
@@ -3,60 +3,55 @@
 		{% include avatar.html class="avatar rounded" %}
 	</div>
 	<div class="col-12 col-md-8 p-0">
-		<div class="card mb-3 card-sm-border-0">
+		<article class="card mb-3 card-sm-border-0">
 			<div class="row m-0 border-bottom">
-			<div class="col-auto d-md-none d-flex p-0">
-			{% include avatar.html width="48px" class="my-auto mr-2" %}
-			</div>
-			<div class="col card-header d-flex justify-content-between pl-1 pl-md-3 border-0">
-				<div>
-				{% capture post_author %}
-				<a href="{{ post.link }}">{{ post.author }}</a>
-				<a href="{{ post.rss }}">{% include icons/feed.svg %}</a>
-				{% endcapture %}
-				{% capture post_language %}
-				{% assign lang = site.localization.locales_set[post.tags.first] %}
-				<a class="text-muted" href="{{ site.baseurl }}/{{ post.tags.first }}/">{{ lang.name }}</a>
-				{% endcapture %}
-				{% capture post_time %}
-				<a class="text-muted" href="{{ post.original_link }}">{{ post.created_at | date: "%H:%M" }}</a>
-				{% endcapture %}
-				<span class="text-muted">
-				{% if include.language %}
-				  {{ locale.postedinat | replace: '$author', post_author | replace: '$language', post_language | replace: '$time', post_time }}
-				{% else %}
-				  {{ locale.postedat | replace: '$author', post_author | replace: '$time', post_time }}
-				{% endif %}
-				</span>
+				<div class="col-auto d-md-none d-flex p-0">
+					{% include avatar.html width="48px" class="my-auto mr-2" %}
 				</div>
-				<div>
-				{% if post.member %}
-					<a title="openSUSE Member" href="{% if post.connect %}https://connect.opensuse.org/pg/profile/{{ post.connect }}{% else %}https://en.opensuse.org/openSUSE:Members{% endif %}" class="badge badge-success"><img src="{{ site.baseurl }}/assets/images/profile/member.svg"/> Member</a>
-				{% elsif post.connect %}
-					<a href="https://connect.opensuse.org/pg/profile/{{ post.connect }}"><img title="Profile on Connect" src="{{ site.baseurl }}/assets/images/profile/connect.svg"/></a>
-				{% endif %}
-				{% if post.irc %}
-				<a title="{{ post.irc }} on Libera" href="irc://irc.libera.chat/{{ post.irc }},isnick" class="badge badge-dark"><img src="{{ site.baseurl }}/assets/images/profile/irc.svg"/> {{ post.irc }}</a>
-				{% endif %}
-				{% if post.gsoc %}
-				<a href="https://summerofcode.withgoogle.com/"><img title="Google Summer of Code" src="{{ site.baseurl }}/assets/images/profile/gsoc.svg"/></a>
-				{% endif %}
```

Comment:
There doesn't seem to be a replacement, and the commit messages don't explain this removal either. Where/how is this information displayed now?